pages/common/xargs.md

```diff
@@ -10,6 +10,10 @@
 
 `{{arguments_null_terminated}} | xargs -0 {{command}}`
 
-- example: list unneeded packages with deborphan and remove them with apt-get
+- example: delete all files that start with 'M'
```

Comment:
Agreed. One can let the description speak for itself without qualifying. I'll edit it along with the `-J` flag example below when I get the OK.